libsolutil/StringUtils.h

```diff
@@ -194,12 +194,10 @@ inline std::string formatNumberReadable(
 		"only integer numbers are supported"
 	);
 
-	if (_value >= 0)
-	{
+	if (_value >= 0) {
 		bigint const _v = bigint(_value);
 		return formatUnsignedNumberReadable(_v, _useTruncation);
-	} else
-	{
+	} else {
```

Comment:
The previous style was alright :)

Reply:
That's also the reason the coding style check failed

Reply:
Fixed